src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -190,6 +241,35 @@ public void parse_validGitRepoLocations_repoConfigurationListCorrectSize() throw
         Assert.assertEquals(2, repoConfigs.size());
     }
 
+    @Test
+    public void parse_repoLocationsAndView_returnsLocationCliArguments() throws ParseException {
+        String input = String.format("-repos \"%s\" %s -view", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertTrue(cliArguments.isAutomaticallyLaunching());
+    }
+
+    @Test
+    public void parse_repoLocationsOnly_success() throws ParseException {
+        String input = String.format("-repos \"%s\" %s", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertFalse(cliArguments.isAutomaticallyLaunching());
+    }
+
+    @Test
+    public void parse_repoLocationsAndViewWithArgs_returnsViewCliArguments() throws ParseException, IOException {
+        String input = String.format("-repos \"%s\" %s -view %s",
```

Comment:
Similar as above. 

`-view` should be a `flag` when used with `-config` and `-repos`.
When used alone, value `PATH` should be expected. `-view=PATH`

Reply:
> When used alone, value `PATH` should be expected. `-view PATH`

How about `java -jar RepoSense.jar -view`? Since running `java -jar RepoSense.jar` by default uses the config files in the cwd, is it reasonable for `java -jar RepoSense.jar -view` to have the same behaviour and in addition automatically launch the report?

Reply:
In this case, it is possible to detect the following invalid command  ->
`java -jar RepoSense.jar -repos REPO_PATH_OR_URL -view /etc/secrets`
because `-repos` does not have a default value like `-config`.

Should I throw an exception in this case? On the other hand we might want the following 2 commands to exhibit similar behaviour:
1. `java -jar RepoSense.jar -repos repo_url -view /etc/secrets`
2. `java -jar RepoSense.jar -config path/to/configs -view /etc/secrets`

Reply:
Sorry didn’t see your other comment:)

Reply:
> How about java -jar RepoSense.jar -view? Since running java -jar RepoSense.jar by default uses the config files in the cwd, is it reasonable for java -jar RepoSense.jar -view to have the same behaviour and in addition automatically launch the report?

Looks fine. Since they are equivalent `java -jar RepoSense.jar -config . -view` <=> `java -jar RepoSense.jar -view` 

> Should I throw an exception in this case? On the other hand we might want the following 2 commands to exhibit similar behaviour:

Don't worry about this comment, it can be done in another PR.